lib/material_input/base_material_input.dart

```diff
@@ -421,12 +432,20 @@ class BaseMaterialInput extends FocusableMixin
     inputRef.nativeElement.select();
   }
 
-  String msgCharacterCounter(int currentCount, int maxCount) => Intl.message(
-      '$currentCount / $maxCount',
-      name: 'BaseMaterialInput_msgCharacterCounter',
-      args: [currentCount, maxCount],
-      desc: 'Character counter shown below a text box in the format "12 / 25"',
-      examples: const {'currentCount': 12, 'maxCount': 25});
+  /// The message to display when character counter is shown.
+  ///
+  /// Always shows [currentCount] but will ignore [maxCount] if it is null.
```

Comment:
How about:
```dart
/// The character count in the form "[currentCount] / [maxCount]"
/// when [maxCount] is non-null; otherwise simply "[currentCount]".
```